tools/file_utils.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+
+def _get_absolut_path():
+    return os.path.abspath(os.curdir)
+
+
+def get_project_path():
+    path = _get_absolut_path()
+
+    if "tools" in path:
+        return path.replace("tools", "")
+
+    return path
+
+
+def get_path_in_project(path_name):
+    return get_project_path() + path_name
```

Comment:
The returned path was not correct on Windows. When I use:
`return os.path.**join**(get_project_path(), path_name)`
it is working ok (creates a path that is platform independent). Please try how does it behave on mac.